test/dummy/app/views/kitchen_sinks/new.html.erb

```diff
@@ -1,5 +1,12 @@
 <h1>Kitchen Sink Demo Page</h1>
 <%= form_for @kitchen_sink, builder: EtGdsDesignSystem.form_builder_class do |f| %>
+
+  <h2>Error Summary</h2>
```

Comment:
You have implemented this in the test page much like you would in a real page.  But, for the rest of the components, they have their own section in the page - is there a reason for this ?